portfolio/src/main/java/com/google/sps/servlets/HelloWorldServlet.java

```diff
@@ -2,17 +2,34 @@
 
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
+import com.google.gson.Gson;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
 /** Handles requests sent to the /hello URL. Try running a server and navigating to /hello! */
 @WebServlet("/hello")
 public class HelloWorldServlet extends HttpServlet {
+    private String[] facts = {"I speak three languages","My go-to deli order is bagel with cream cheese & jelly","My Favorite book is Things Fall Apart by Chinua Achebe"};
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    String json = convertToJson(facts);
     response.setContentType("text/html;");
-    response.getWriter().println("<h1>Hello world!</h1>");
+    response.getWriter().println(json);
   }
-}
+
+   private String convertToJson(String[] facts) {
```

Comment:
You can use the GSON library to do the Java Array -> JSON conversion for you:

```java
Gson gson = new Gson();
String[] example = {
  "foo",
  "bar",
  "baz"
};
String json = gson.toJson(example);
```
https://sites.google.com/site/gson/gson-user-guide